gui/wxpython/psmap/frame.py

```diff
@@ -1049,7 +1049,7 @@ def makePSFont(self, textDict):
             fn = wx.Font(
                 pointSize=fontsize, family=family, style=style, weight=weight, face=face
```

Comment:
Looks like the parameters need to be updated:
```suggestion
                pointSize=fontsize, family=family, style=style, weight=weight, faceName=face
```
see https://docs.wxpython.org/wx.Font.html#wx.Font.__init__